client/lib/load-script/README.md

```diff
@@ -4,8 +4,9 @@ Async Script Loader
 This utility function allows us to use a standardized method of loading remote scripts and injecting them into the `<head>` of our document to bring external functionality into the app.
 
 ### Usage
-```js
-loadScript = require( 'lib/load-script' );
+```es6
```

Comment:
I noticed this week that GitHub no longer highlights es6 - I've not found any information on this but my guess is that this is because ` ```js` now highlights es6 syntax properly.
I'll make sure that this PR gets this fixed before merge... and then I'll look at re-tagging instances of ` ```es6` :)